tests/test_basic.py

```diff
@@ -153,7 +154,7 @@ def test_disallow_string_for_allowed_methods(app):
 
 
 def test_url_mapping(app, client):
-    random_uuid4 = "7eb41166-9ebf-4d26-b771-ea3f54f8b383"
+    random_uuid4 = uuid.uuid4()
```

Comment:
You should consider to convert UUID to string.

If you don't convert the UUID to a string explicitly, it might cause issues later in the code if a string representation of the UUID is expected. The suggested improvement makes the intent clear.

If the code later expects a string representation of the UUID, not converting it to a string might lead to unexpected errors or incorrect behavior.

In addition, forcing the name of variable types makes the code more readable and reduces confusion